### PR TITLE
Add example of using mongoose-sequence

### PR DESCRIPTION
**Pull request by dandv**

### CODE REVIEW

File: github-page/_guides/advanced/plugins.md [hunk 1]
```diff
@@ -64,6 +64,21 @@ const SomeClassModel = getModelForClass(SomeClass);
...
+## mongoose-sequence
+
+To use [mongoose-sequence](https://github.com/ramiel/mongoose-sequence), import the plugin and use it like this:
+```ts
+import AutoIncrementFactory from 'mongoose-sequence'; 
+
+// AutoIncrement now is the instance
+const AutoIncrement = AutoIncrementFactory(mongoose);
+
+@plugin(AutoIncrement, { inc_field: 'id', start_seq: 200 })
```
Comment (hasezoey) [inline github-page/_guides/advanced/plugins.md:76]: are you sure it should be `id` and not `_id`?

File: github-page/_guides/advanced/plugins.md [hunk 2]
```diff
@@ -64,6 +64,21 @@ const SomeClassModel = getModelForClass(SomeClass);
...
+class ... { ... }
```
Comment (hasezoey) [inline github-page/_guides/advanced/plugins.md:77]: shouldnt the `inc_field` be an type in the class?

**Comment by hasezoey** [comment]
@dandv could you update your pr?

**Comment by dandv** [comment]
Hi, I wonder if this PR is useful any more, given you've been writing a separate [autoincrement plugin](https://github.com/typegoose/auto-increment)?

**Comment by hasezoey** [comment]
@dandv as long as the plugin works with typegoose, it can be added - i dont care if its an competitor to an plugin from typegoose

**Pull request merged**